source/fundamentals/connection.txt

```diff
@@ -247,6 +253,12 @@ URI to specify the behavior of the client.
      - Specifies whether to serialize functions on any object passed to the
        server.
 
+   * - **serverApi**
+     - ``string`` or ``enum``
```

Comment:
Suggestion: Because no other row has their type monospaced, it makes this row stand out. Either the types column needs to all be monospaced, or the monospace needs to be removed here.